HotelWebsite/css/style.css

```diff
@@ -0,0 +1,301 @@
+/* first reset the margin and padding */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* main styling */
+html,
+body {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    line-height: 1.7em;
+
+}
+
+a {
+    color: #333;
+    text-decoration: none;
+}
+
+h1,
+h2,
+h3 {
+    padding: 20px;
+}
+
+p {
+    margin: 10px 0;
+
+}
+
+/* utility classes */
+/* these are the classes that we will use throught the 
+site and are not specific to any specific area*/
+
+.container {
+    margin: auto;
+    max-width: 1100px;
+    overflow: auto;
+    padding: 0 20px;
+}
+
+.primary-heading {
+    color: #f7c08a;
+}
+
+.lead {
+    font-size: 18px;
+
+}
+
+.btn {
+    display: inline-block;
+    font-size: 18px;
+    color: #fff;
+    background: #333;
+    padding: 13px 20px;
+    border: none;
+    cursor: pointer;
+
+}
+
+.btn:hover {
+    background: #f7c08a;
+    color: #333;
+}
+
+.btn-light {
+    background: #f4f4f4;
+    color: #333;
+    margin-top: 20px;
+}
+
+.clr {
+    clear: both;
+}
+
+.l-heading {
+    font-size: 40px;
+}
+
+/* paddind for top and bottom */
+.py-1 {
+    padding: 10px 0;
+}
+
+.py-2 {
+    padding: 20px 0;
+}
+
+.py-3 {
+    padding: 30px 0;
+}
+
+/* .bg-dark {
+    background: #333;
+    color: #fff;
+} */
+
+/* .bg-light {
+    background: #f4f4f4;
+    color: #333;
+}
+
+.bg-primary {
+    background: #333;
+    color: #f7c08a;
+} */
+
+/*  navbar styling */
+#navbar {
+    background: #333;
+    color: #fff;
+    overflow: auto;
+}
+
+#navbar a {
+    color: white;
+}
+
+#navbar h1 {
+    float: left;
+    padding: 20px;
+    color: white;
+}
+
+#navbar ul {
+    list-style: none;
+    float: right;
+}
+
+#navbar ul li {
+    float: left;
+}
+
+#navbar ul li a {
+    display: block;
+    padding: 20px;
+    text-align: center;
+}
+
+#navbar ul li a:hover,
+#navbar ul li a.current {
+    background: #444;
+    /* border-style: solid; */
+    /* border-radius: 8px; */
+    color: #f7c0aa;
+
+}
+
+/* showcase */
+
+#showcase {
+    background: url(../img/backG.jpg) no-repeat center center/cover;
+    height: 600px;
+
+}
+
+#showcase .showcase-content {
+    color: #fff;
+    text-align: center;
+    padding-top: 170px;
+}
+
+#showcase .showcase-content h1 {
+    font-size: 60px;
+    line-height: 1.2em;
```

Comment:
Read difference between em/rem/vw-vh etc

Reply:
em- where 'm' unit is a multiplier or relative to the font size of its parent container or its parent element.
rem- it is relative to the font size of the root HTML element.
VH & VW- H stands for the height and W stands for the width , our page always gonna have 100 viewports height and 100 viewports width no matter what's the size of web page